parsing/dialects/intermediate/dialects/dialect1/parserImpls.ftl

```diff
@@ -4641,6 +4641,11 @@ SqlNode CreateProcedureStmt() :
         e = ConditionalStmt()
     |
         e = CursorStmt()
+    |
+        // This lookahead ensures parser chooses
+        // the right path when facing begin label.
```

Comment:
nit: Seems like this line was wrapped to soon and more can fit on the previous line.

Reply:
line wrap fixed in https://github.com/googleinterns/calcite/pull/280/commits/b9f8fa4c7b3b2c5379d302ff26031064b0ed9af9